Fix "content management issues"

maximiliankolb opened the pull request:
#### What changes are you introducing?

Fixes for https://github.com/theforeman/foreman-documentation/issues/3475

#### Why are you introducing these changes? (Explanation, links to references, issues, etc.)

#### Anything else to add? (Considerations, potential downsides, alternative solutions you have explored, etc.)

#### Checklists

* [ ] I am okay with my commits getting squashed when you merge this PR.
* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.13/Katello 4.15
* [x] Foreman 3.12/Katello 4.14 (Satellite 6.16)
* [x] Foreman 3.11/Katello 4.13 (orcharhino 6.11 on EL8 only)
* [ ] Foreman 3.10/Katello 4.12
* [ ] Foreman 3.9/Katello 4.11 (Satellite 6.15; orcharhino 6.8/6.9/6.10)
* [ ] Foreman 3.8/Katello 4.10
* [ ] Foreman 3.7/Katello 4.9 (Satellite 6.14)
* We do not accept PRs for Foreman older than 3.7.

Lennonka reviewed (commented):
LGTM, but I've found an additional issue, see https://github.com/theforeman/foreman-documentation/issues/3475#issuecomment-2549629448

maximiliankolb commented:
> LGTM, but I've found an additional issue, see [#3475 (comment)](https://github.com/theforeman/foreman-documentation/issues/3475#issuecomment-2549629448)

Rebased to "master" and fixed both additional issues. Also converted the upstream URLs to "https".

Lennonka commented on guides/common/modules/ref_best-practices-for-patching-content-hosts.adoc:
```diff
@@ -1,7 +1,7 @@
 = Best practices for patching content hosts
 
-* Registering hosts to {Project} requires {project-client-name}, which contains the `subscription-manager` package, `katello-host-tools` package, and their dependencies.
+* Registering hosts to {Project} requires {project-client-name}, which contains the `katello-host-tools-tracer` package and its dependencies.
```
```suggestion
* Registering hosts to {Project} requires {project-client-name}, which contains the `katello-host-tools` package and its dependencies.
```
It contains both. I'm not sure about changing the package name, but it isn't a blocker.

maximiliankolb replied:
Fixed.

Lennonka reviewed (approved):
Thank you for these fixes! LGTM

maximiliankolb commented:
Merged to "master" and cherry-picked:
   df13921275..eb2cfa7aab  3.13 -> 3.13
   af2173ddcb..498cd503b0  3.12 -> 3.12
